_posts/en/newsletters/2020-02-26-newsletter.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Bitcoin Optech Newsletter #86'
+permalink: /en/newsletters/2020/02/26/
+name: 2020-02-26-newsletter
+slug: 2020-02-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes two proposed routing improvements for
+LN, summarizes three interesting talks from the Stanford Blockchain
+Conference, links to popular questions and answers from the Bitcoin
+StackExchange, and lists several notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.  Note: the list of releases and release candidates has
+been moved to its own [section][release rc section].*
+
+## News
+
+- **Reverse up-front payments:** as described in [Newsletter #72][news72
+  upfront], LN developers have been looking for a way to charge a small
+  fee for routing an LN payment (HTLC) even if the payment is rejected.
+  This can discourage payments that are designed to consume bandwidth
+  and liquidity before ultimately failing at no cost to the sender.
+  This week, Joost Jager [proposed][jager up-front] a new scheme where
+  up-front fees are paid from the receiver of an HTLC back towards the
```

Comment:
s/towards/to/

Reply:
I think towards is better because the fees are not necessarily paid **to** the spender but in their general direction (**towards**).